src/runtime/runtime-test.js

```diff
@@ -184,22 +228,6 @@ describes.realWin('installRuntime', (env) => {
     await getRuntime().whenReady();
     expect(progress).to.equal('123');
   });
-
-  it('implements all APIs', async () => {
```

Comment:
Note: This test was essentially a copy of the "should implement Subscriptions interface" test